x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -220,24 +313,33 @@ export class RemoteClusterFormUi extends Component {
     this.onFieldsChange({ seeds: seeds.map(({ label }) => label) });
   };
 
-  renderSeeds() {
+  getCurrentFields() {
     const {
-      areErrorsVisible,
       fields: {
-        seeds,
-      },
-      fieldsErrors: {
-        seeds: errorsSeeds,
+        transientSettings,
+        persistentSettings,
       },
-      localSeedErrors,
+      currentSettingType,
     } = this.state;
 
+    return currentSettingType === 'transient' ? transientSettings : persistentSettings;
+  }
+
+  clearCurrentFields = () => {
+    this.onFieldsChange({
+      seeds: [],
+      skipUnavailable: null,
+    });
+  }
+
+  renderSeeds() {
+    const { localSeedErrors } = this.state;
     const { intl } = this.props;
+    const { seeds } = this.getCurrentFields();
 
-    // Show errors if there is a general form error or local errors.
```

Comment:
I think this change broke the UX slightly. These conditions ensured that hitting "Save" when no seeds have been entered will show the validation message that seeds are required. With these changes, this validation doesn't appear.